vnext/flow-typed/npm/react-dom_v16.x.x.js

```diff
@@ -0,0 +1,115 @@
+/**
```

Comment:
Just a generic comment. Shouldn't flow adopt the same pattern that TypeScript has by adding @flow-typed namespace to npm?

Reply:
I do think they have packages like that. Some of these differ from the public flow-typed packages though it looks like. So RN itself internally has its own typings stored in the file tree that its using that we want to replicate so we do type-checking of our sources in the same way.